### ISSUE TITLE
When going back from message list, show sidebar menu instead of closing the app

### ISSUE THREAD

**Issue by ontheair81** [thundernest/k-9#7018]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same feature request.
- [X] I will describe the problem with as much detail as possible.
- [X] This issue only contains a request for one single feature, **not** multiple (related) features.

### App version

6.602

### Problem you are trying to solve

When reading the message list, pushing the "back navigation button" of Android always exits the app immediately, bringing the user back to the launcher.

The only way going back to the sidebar (containing the folder view and account view) is the hamburger menu in a not-so-easy to reach position (top left, the worst possible position for right-handed users).

### Suggested solution

When pressing the "back navigation button", the sidebar should be shown instead of closing the app.

**Comment by cketti** [comment]
The current behavior is the recommended pattern on Android and how pretty much all other email apps work. If you want to open the drawer using a gesture, put down your finger at the edge of the screen, hold it for a moment, then drag it to the middle of the screen.

**Comment by ontheair81** [comment]
Thank you for your feedback!

> The current behavior is the recommended pattern on Android and how pretty much all other email apps work.

Ok, I did not know that. The apps that I used most often (a RSS reader and the previous email app before I switched to K-9) opened the drawer by using the back button. Of course I will train my muscle memory now this different way ;-)

Just wanted to point out that my suggested solution could improve ergonomics and workflow especially for people like me, using email all the day - where a gesture is less accurate (and could mark an email mistakenly) than pushing a button. So, having an option to create that behaviour still would make sense in my opinion.

Just my humble opinion and just a suggestion, of course.
Thank you for all the work on K-9 Mail!
